build.gradle

```diff
@@ -33,13 +33,17 @@ dependencies {
     compile group: 'commons-io', name: 'commons-io', version: '2.4'
     compile group: 'org.apache.commons', name: 'commons-collections4', version: '4.0'
     compile group: 'com.google.code.gson', name: 'gson', version: '2.3.1'
+    compile group: 'com.google.guava', name: 'guava', version: '18.0'
     compile group: 'com.rabbitmq', name: 'amqp-client', version: '3.4.4'
     compile group: 'org.apache.logging.log4j', name: 'log4j-api', version: '2.1'
     compile group: 'org.apache.logging.log4j', name: 'log4j-core', version: '2.1'
     compile group: 'com.codepoetics', name: 'protonpack', version: '1.2'
     compile group: 'org.eclipse.jetty', name: 'jetty-server', version: '9.2.1.v20140609'
     compile group: 'org.eclipse.jetty', name: 'jetty-servlet', version: '9.2.1.v20140609'
-    compile group: 'net.sf.trove4j', name: 'trove4j', version: '3.0.3'
+    compile group: 'net.sf.trove4j', name: 'trove4j', version: '2.0.2'
```

Comment:
Yes. Mallet still uses 2.0.2, and I couldn't figure out how to get both to co-exist in the build.